charts/dapr/charts/dapr_operator/values.yaml

```diff
@@ -15,6 +15,8 @@ fullnameOverride: ""
 
 runAsNonRoot: true
 
+args: []
```

Comment:
What's this for?

Reply:
this is for totally arbitrary flags for each components, was my first approach before to have something specific under global, I think that still useful for greater flexibility, actually not used by default

Reply:
If it's currently unused, I'd remove it, just to avoid causing confusion